pages/article.md

```diff
@@ -15,7 +15,7 @@ By Princess Ojiaku , Edwin Torres , Mark Headd
 
 April 1, 2021
 
```

Comment:
Just remembered I may have already done this in another open PR. Probably fine to merge this in. Just be aware there may be a conflict to address.

Reply:
@pglevy, I see the branch you created (article-tags-front-matter). I can create a PR, review, and merge 👍🏻